modules/client-configuration/pages/clients-oracle.adoc

```diff
@@ -0,0 +1,106 @@
+====
+endif::[]
+
+
+[IMPORTANT]
+====
+Traditional clients are not available on {oracle}{nbsp}8.
+{oracle}{nbsp}8 clients are only supported as Salt clients.
+====
+
+
+
+== Channel and Repository Management
+
+ifeval::[{suma-content} == true]
+.Procedure: Adding Client Tools Channels
+. On the {productname} Server, add the appropriate {oracle} channels:
++
+* For {oracle} 6:
++
+From the {webui}, add [systemitem]``Oracle Linux 6 x86_64``.
++
+From the command prompt, add [systemitem]``oraclelinux6-x86_64``.
++
+* For {oracle} 7:
++
+From the {webui}, add [systemitem]``Oracle Linux 7 x86_64``.
++
+From the command prompt, add [systemitem]``oraclelinux7-x86_64``.
++
+* For {oracle} 8:
++
+From the {webui}, add [systemitem]``Oracle Linux 8 x86_64``.
++
+From the command prompt, add [systemitem]``oraclelinux8-x86_64``.
+. Synchronize the {productname} Server with the {SCC}.
+You can do this using the {webui}, or by running [command]``mgr-sync`` at the command prompt.
+. Add the new channel to your activation key.
+endif::[]
+
+
+ifeval::[{uyuni-content} == true]
+[IMPORTANT]
+====
+For {oracle} 8 clients, add both the ``BaseOS`` and ``Appstream`` channels.
+You will require packages from both channels.
+If you do not add both channels, you will not be able to create the bootstrap repository, due to missing packages.
+====
+
+
+.Procedure: Adding Client Tools Channels
+. At the command prompt on the {productname} Server, as root, install the [package]``spacewalk-utils`` package:
++
+----
+zypper in spacewalk-utils
+----
+. Add the {oracle} base, updates, and client channels, specifying the {oracle} version and architecture:
++
+----
+spacewalk-common-channels -a x86_64 oraclelinux8 \
+oraclelinux8-uyuni-client oraclelinux8-appstream
+----
+
+
+[NOTE]
+====
+The client tools channel provided by [command]``spacewalk-common-channels`` is sourced from {uyuni} and not from {suse}.
+====
+
+endif::[]
+
+
+There are two ways to check if a channel has finished synchronizing:
+
+* In the {productname} {webui}, navigate to menu:Admin[Setup Wizard] and select the [guimenu]``Products`` tab.
++
+This dialog displays a completion bar for each product when they are being synchronized.
+* Check the synchronization log file at the command prompt with [command]``tail -f /var/log/rhn/reposync/channel-label.log`` file.
++
+Each child channel will generate its own log during the synchronization progress.
+You will need to check all the base and child channel log files to be sure that the synchronization is complete.
+
+
+
+== Create an Activation Key
+
+You will need to create an activation key that is associated with your {oracle} channels.
+
+For more information on activation keys, see xref:client-configuration:clients-and-activation-keys.adoc[].
+
+
+
+== Register Clients
+
+{oracle} clients are registered in the same way as all other clients.
+For more information, see xref:client-configuration:registration-overview.adoc[].
```

Comment:
We need to add information about Oracle not trusting Uyuni packages, as we have it for CentOS: https://www.uyuni-project.org/uyuni-docs/uyuni/client-configuration/clients-centos.html#_trust_gpg_keys_on_clients

This is (or should be) on all the client pages now.